src/scss/_packages.scss

```diff
@@ -2,80 +2,172 @@
-                        @apply text-sm truncate overflow-ellipsis;
+                .title-link {
+                    @apply text-lg;
 
-                        width: 300px;
+                    &:hover {
+                        @apply text-blue-700 underline;
                     }
                 }
-            }
-        }
 
-        @include transition;
+                pulumi-accordion {
+                    $svgDefaultStrokeColor: #000000;
 
-        &:hover {
-            @apply shadow-md;
-            transform: scale(1.03);
-        }
+                    pulumi-accordion-item {
+                        @apply rounded;
 
-        .package-tags {
-            .package-tag:not(last-child) {
-                @apply mr-4 p-2;
-            }
+                        border: 2px solid #131314;
 
-            .package-category {
-                background-color: map-get($blue, 200);
+                        &:hover {
+                            @apply bg-black;
 
-                &.network {
-                    background-color: map-get($yellow, 200);
+                            .heading {
+                                @apply text-white;
+                            }
+
+                            svg {
+                                stroke: #FFFFFF;
+                            }
+                        }
+
+                        // Reset the background of the accordion item
+                        // when it is expanded.
+                        &.expanded {
+                            @apply bg-transparent;
+
+                            .heading {
+                                color: inherit;
+                            }
+
+                            svg {
+                                stroke: $svgDefaultStrokeColor;
+                            }
+                        }
+
+                        .heading, ul {
+                            @apply text-base;
+                        }
+
+                        ul {
+                            @apply list-none;
+                        }
+
+                        svg {
+                            stroke: $svgDefaultStrokeColor;
+                        }
+                    }
                 }
-            }
 
-            .native-package {
-                background-color: map-get($purple, 400);
-            }
+                .description {
+                    @apply overflow-ellipsis overflow-hidden;
 
-            .package-status {
-                &.ga, &.public-preview {
-                    background-color: map-get($fuchsia, 200);
+                    // The following is non-standard, but is weirdly supported
+                    // by all major browsers with the -webkit- prefix.
+                    // https://caniuse.com/?search=line-clamp
+                    display: -webkit-box;
+                    // Show ellipsis after 2 lines.
+                    -webkit-line-clamp: 2;
+                    -webkit-box-orient: vertical;
```

Comment:
What is the intent for this? I'm not sure I understand.

Reply:
It's to apply ellipsis after two lines. The normal ellipsis is a single-line ellipsis. Without this we could have a component that does the truncation after a certain number of lines if we don't want to use this. But this does work in all major browsers despite the `-webkit` prefix.